### PR TITLE
Update to resources.json

### PR DESCRIPTION
**Pull request by deniseallela**
Change on about GDG and addition keyword

### CODE REVIEW

File: data/resources.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   "aboutOrganizers": "<a href='http://lviv.gdg.org.ua' target='_blank'>GDG Lviv</a> with support of 10 GDGs from all around Ukraine is proud organizer of DevFest Ukraine",
   "openTeamPage": "Open team page",
   "whatIsGdg": "What is GDG?",
-  "aboutGdg": "Google Developers Group (GDG) - is open and volunteer geek community who create exciting projects and share experience about Google technologies with a passion",
+  "aboutGdg": "Google Developers Group (GDG) - is open community of volunteers who come up with creating and exciting ideas with the aim of coming up with solutions to day to day problems within the society.(both locally and globally)",
```
Comment (sigu) [inline data/resources.json:38]: Is this creative or creating? 🤔

File: data/resources.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "title": "GDG DevFest Season 2017",
   "description": "The biggest Google tech conference in Kenya carefully crafted for you by the GDG community (GDG Kisumu, GDG Rongo, GDG Kisii and GDG MMUST)! All about Android, Web and Cloud from the world experts",
-  "keywords": "event, gdg, gde, devfest, google, programming, android, kisumu,  chrome, polymer, developers, web, cloud, androiddev",
+  "keywords": "event, gdg, gde, devfest, google, programming, android, kisumu,  chrome, polymer, developers, web, cloud, androiddev , iot",
```
Comment (sigu) [inline data/resources.json:4]: How did we forget about IOT, ongeza hadi arduino 😛

**Review by sigu — COMMENTED** [review]
Hey, check on the comments

File: data/resources.json (hunk 2 above)
Comment (deniseallela) [reply data/resources.json:4]: Let me do so just now...

File: data/resources.json (hunk 1 above)
Comment (deniseallela) [reply data/resources.json:38]: it should be creative and the beginning part should be is an open not  is open...let me fix that

**Comment by sigu** [comment]
@deniseallela can we have this sorted and merged tomorrow?

**Review by sigu — COMMENTED** [review]
See comments

**Comment by sigu** [comment]
replaced by PR #2

**Pull request closed without merging**